app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, class_name: 'User', foreign_key: :post_id
+
+  def update_like_counter(id)
+    new_counter = Like.where(post_id: id).count
+    Post.where(id: id).update(likes_counter: new_counter)
+  end
+end
```

Comment:
- [ ] kindly work on all the methods that are performing increments, passing a parameter to a method that will not be used elsewhere becomes unnecessary. We can also make use of the active record callback methods like `after_save` or `after_update`, and hide our `update_likes_counter` method under private, something like so:
```
after_save: update_likes_counter

private
def update_likes_counter
    author.increment!(:likes_counter)
 end
```